ws-nextjs-app/pages/[service]/send/[id]/UGCPageLayout.tsx

```diff
@@ -14,29 +14,36 @@ const UGCPageLayout = ({ pageData }: PageProps) => {
   const { title, description, sections } = pageData;
 
   const { fields } = sections?.[0] ?? {};
+  const sectionTitle = sections?.[0].sectionText?.title ?? '';
 
   return (
     <>
       <Metadata
-        title="Test UGC Form"
+        title={title}
         lang={lang}
         description="Test UGC Form"
         openGraphType="website"
         hasAmpPage={false}
       />
-      <div css={styles.grid}>
-        <div css={styles.primaryColumn}>
-          <main css={styles.mainContent}>
-            <Heading level={1}>{title}</Heading>
-            <div
-              // TODO: This is a security risk, we should sanitize the HTML
-              // eslint-disable-next-line react/no-danger
-              dangerouslySetInnerHTML={{ __html: description }}
-            />
-            <FormContextProvider fields={fields}>
-              <Form fields={fields} />
-            </FormContextProvider>
-          </main>
+      <div css={styles.background}>
+        <div css={styles.gradient} />
+        <div css={styles.grid}>
+          <div css={styles.primaryColumn}>
+            <main css={styles.mainContent}>
+              <Heading level={1}>{title}</Heading>
+              <div
+                // TODO: This is a security risk, we should sanitize the HTML
+                // eslint-disable-next-line react/no-danger
+                dangerouslySetInnerHTML={{ __html: description }}
```

Comment:
Should this be fixed as part of this PR?

Reply:
We should evaluate the security risk around using it like this and see if it warrants parsing out the string into HTML elements in the BFF.

It will need styled though if its kept like this. Doing it this way means we aren't using our `Paragraph` components, so it will need manual styling to avoid it using the browser default.

Reply:
So this probably needs a seperate subtask under [WSTEAM1-969](https://jira.dev.bbc.co.uk/browse/WSTEAM1-969)